### PR TITLE
[Docs] add community handbook

### PR DESCRIPTION
**Pull request by sunshinemingo**
Community handbook v1.0

### CODE REVIEW

File: community_meeting_record.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+time: 2020.4.15
+host: sunshinemingo
+attendee: [sunshinemingo, wengzhenjie, gymgym1212, zhuxiangning, AnAsteria, will-ww, xiaoya-Esther, yangming, birdflyi, JingbenShi668, tedliu1]
+agenda:
+  - content: New data description
+    proposer: frank-zsy
+    participants: [sunshinemingo]
+  - content: Design of credit system
+    proposer: frank-zsy
```
Comment (frank-zsy) [inline community_meeting_record.yaml:9]: The proposer of this topi is @gymgym1212 I think

File: OpenDigger_Community_Handbook.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+time: XXX.XX.XX
+host: GitHub ID
+attendee: [GitHub ID1, GitHub ID2, GitHub ID3, GitHub ID4]
+agenda:
+	- content: Topic Description1
+		proposer: GitHub ID
+		participants: [GitHub ID1, GitHub ID2, GitHub ID3]  
+	- content: Topic Description2 
+		proposer: GitHub ID 
+		participants: [GitHub ID1, GitHub ID2, GitHub ID3]
```
Comment (frank-zsy) [inline OpenDigger_Community_Handbook.md:49]: Did you use tab for indentation? Only white space can be used in YAML file format I think.

File: OpenDigger_Community_Handbook.md [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+
+* Send a reminder about community meetings
+* Prepare weekly community meeting
+* Host weekly community meeting
+* Make sure meeting minutes are kept
+* Collect themes from a week in the OpenDigger community
+* The host can be changed without any punishment before 4 pm every Thursday
+* The host can be changed by deducting the credits that the hostshould getone hour before the meeting
+* Count credits of participants in the weekly meeting
+    * Format: **yaml**
```
Comment (frank-zsy) [inline OpenDigger_Community_Handbook.md:34]: Maybe we can add [YAML](https://yaml.org/) link here.

File: community_meeting_record.yaml (hunk 1 above)
Comment (sunshinemingo) [reply community_meeting_record.yaml:9]: Yes, I think that is more reasonable.

File: OpenDigger_Community_Handbook.md (hunk 3 above)
Comment (sunshinemingo) [reply OpenDigger_Community_Handbook.md:34]: OK

**Pull request closed without merging**